templates/eap72-openjdk11-image-stream.json

```diff
@@ -111,6 +111,69 @@
                     }
                 ]
             }
+        },
+        {
+            "kind": "ImageStream",
+            "apiVersion": "v1",
+            "metadata": {
+                "name": "jboss-eap72-openj9-11-openshift-rhel8",
+                "annotations": {
+                    "openshift.io/display-name": "Red Hat JBoss EAP 7.2 OpenJ9 11 UBI 8 Image",
```

Comment:
Should it be "UBI 8 Image" or "RHEL 8 Image"?

Reply:
The RCM default is to just use -rhel7 / -rhel8 so there aren't too many variants of the names.

Reply:
Ping for approval so I can merge @jmesnil :)